163519119 comment track edit history

NkFab opened the pull request:
#### What does this PR do?
Have comment track comment history.

#### Description of Task to be completed?
When a logged-in user edits a comment, it should show that the comment was edited and be able to see the original comment.

#### How should this be manually tested?
- Clone the repo
- `npm install` or `yarn` to install dependencies
- `npm run start` or `yarn start` to start the app 
- View an article and add a comment
- Edit the comment by clicking the edit icon on the right top corner.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#163519119

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/22955494/57143524-54c6b300-6dbf-11e9-9911-73ddcf833348.png)


#### Questions:

oesukam committed:
feat: implement article's rating

oesukam committed:
feat: add article's rating page

NkFab committed:
feat: implement share article on social platforms [Starts #163519109]

NkFab committed:
feat: add unit tests [Finishes #16351909]

NkFab committed:
feat: add unit tests [Finishes #16351909]

Nziranziza committed:
(comment) CRUD comment
- add comment reducer
- add comment action
- add comment component
- add commentRender component
[starts #163519104]

Nziranziza committed:
(comment) CRUD comment
- add comment reducer
- add comment action
- add comment component
- add commentRender component
[starts #163519104]

Nziranziza committed:
(comment) CRUD comment
- add comment reducer
- add comment action
- add comment component
- add commentRender component
[starts #163519104]

Nziranziza committed:
(comment) CRUD comment
- add comment reducer
- add comment action
- add comment component
- add commentRender component
[starts #163519104]

Nziranziza committed:
(comment) CRUD comment
- add comment reducer
- add comment action
- add comment component
- add commentRender component
[starts #163519104]

Nziranziza committed:
(comment) CRUD comment
- add comment reducer
- add comment action
- add comment component
- add commentRender component
[starts #163519104]

Nziranziza committed:
(comment) CRUD comment
- add comment reducer
- add comment action
- add comment component
- add commentRender component
[starts #163519104]

mcaleb808 committed:
feat(articles): adds tags filtering in article view

- add tests
- add tags filtering in article component

[finish #163519108]

oesukam committed:
feat: add actions and reducers

oesukam committed:
feat: add slide's styles

oesukam committed:
feat: change slide navigation style

Nziranziza committed:
feat(comment) CRUD comment
- add more test
[finishes #163519104]

Nziranziza committed:
feat(comment) CRUD comment
- add more test
[finishes #163519104]

Nziranziza committed:
feat(comment) CRUD comment
- add more test
[finishes #163519104]

Nziranziza committed:
feat(comment) CRUD comment
- add more test
[finishes #163519104]

Nziranziza committed:
feat(comment) CRUD comment
- add more test
[finishes #163519104]

Nziranziza committed:
feat(comment) CRUD comment
- add more test
[finishes #163519104]

Nziranziza committed:
mend

Nziranziza committed:
mend

Nziranziza committed:
mend

Nziranziza committed:
mend

Nziranziza committed:
mend

Nziranziza committed:
mend

Nziranziza committed:
mend

filschristian committed:
feat(like/dislike comment): add like and dislike icons
- add icons
- add styles

[Starts #163519120]

filschristian committed:
feat(like/dislike comment): add like and dislike icons
- add icons
- add styles

[Starts #163519120]

mcaleb808 committed:
feat(home): add unit tests

mcaleb808 committed:
feat(home): load more articles on scroll

mcaleb808 committed:
feat(home): position article card

mcaleb808 committed:
feat(home): add login to Navbar

filschristian committed:
feat(like/dislike comment): add actions and view like and dislike

filschristian committed:
feat(like/dislike comment): add actions and view like and dislike

NkFab committed:
feat: implement comment track history [Starts #163519119]

NkFab committed:
feat: implement comment track history [Starts #163519119]

filschristian committed:
feat(like/dislike comment): add unit tests

mcaleb808 committed:
feat(tags): implement feedback

filschristian committed:
feat(like/dislike comment): add unit tests

filschristian committed:
feat(like/ dislike comment): implement feeback

filschristian committed:
feat(like/ dislike comment): implement feeback

mcaleb808 committed:
feat(tags): rebase develop

filschristian reviewed (changes requested):
Please work on the comments.
Please change the title of this PR

mcaleb808 committed:
feat(home): add home and landing pages

- add tests
- add Home Components
- add Home page functionalities
- add Landing page functionalities

[finishes 165571064]

mcaleb808 committed:
feat(home): fix tests

NkFab committed:
feat: imlement feedback

NkFab committed:
feat: imlement feedback

mcaleb808 committed:
feat(Home): fix reducers tests

mcaleb808 committed:
feat(Home): fix reducers tests

mcaleb808 committed:
feat(Home) add home tests

kimotho-njoki committed:
Merge pull request #19 from andela/ft-share-article-social-163519109

#163519109  share article social

Nziranziza committed:
feat(comment)
- rebase

kimotho-njoki committed:
Merge pull request #21 from andela/ft-CRUD-comment--article-163519104

#3163519104 crud comment  article

mcaleb808 committed:
feat(tags): rebase develop"

kimotho-njoki committed:
Merge pull request #22 from andela/ft-navigate-articles-with-tags-163519108

#163519108 Implement filter by tags when viewing an article

filschristian committed:
feat(notifications): first commit
- add notification icon
- add notification component
- add action creators
- handle actions in reducers

[Starts #163519111]

filschristian committed:
feat(like/dislike comment): rebase and fix conflicts

filschristian committed:
feat(notifications): add all notification view page

filschristian committed:
feat(notifications): add unit tests

filschristian committed:
feat(notifications): fix conflicts

filschristian committed:
feat(notifications): add unit tests

kimotho-njoki committed:
Merge pull request #23 from andela/ft-like-dislike-comment-163519120

#163519120 Add like and dislike a comment

kimotho-njoki committed:
Merge pull request #27 from andela/ft-user-view-notifications-163519111

#163519111 Add Notifications

mcaleb808 committed:
feat(home): rebase develop

kimotho-njoki committed:
Merge pull request #25 from andela/ft-landing-page-165571064

#165571064 Add landing and home pages

NkFab committed:
Merge branch 'ft-comment-track-edit-163519119' of github.com:andela/Lit-Kigali1-Frontend into ft-comment-track-edit-163519119

NkFab committed:
Merge branch 'ft-comment-track-edit-163519119' of github.com:andela/Lit-Kigali1-Frontend into ft-comment-track-edit-163519119

NkFab committed:
Merge branch 'ft-comment-track-edit-163519119' of github.com:andela/Lit-Kigali1-Frontend into ft-comment-track-edit-163519119
